### PR TITLE
Allow filling the `assigned_to` field on event_add view

### LINKED ISSUE #847: Assign admin when creating new event
You should be able to assign admin from this page:
https://amy.software-carpentry.org/workshops/events/add/

Right now you can't assign admin until you save and edit.

### PR DESCRIPTION
**Pull request by narayanaditya95**
Fixes #847.

### CODE REVIEW

**Comment by pbanaszkiewicz** [comment]
That's even easier than I thought!

There should be a test for adding a new event, so please add check for the `assigned_to` in there. If there isn't, can you please create it? Thanks!

**Comment by narayanaditya95** [comment]
@pbanaszkiewicz Done. Also verified that the test fails without the fix.

File: workshops/test/test_event.py [hunk 1]
```diff
@@ -328,6 +328,24 @@ def test_unique_slug(self):
...
+            'assigned_to': self.admin.pk,
+            'invoice_status': 'unknown',
+        }
+        response = self.client.post(reverse('event_add'), data, follow=True)
+        event = Event.objects.get(slug='2016-07-09-test')
+        self.assertRedirects(
+            response,
+            reverse('event_details', kwargs={'event_ident': event.slug}),
+        )
+        self.assertEqual(response.context['event'].assigned_to, self.admin)
```
Comment (pbanaszkiewicz) [inline workshops/test/test_event.py]: Please change that to `event.refresh_from_db()` and then assert on that `event.assigned_to`.

Comment (narayanaditya95) [inline workshops/test/test_event.py]: Why do we have to `event.refresh_from_db()` when we are fetching the event object after a POST?
`self.assertEqual(event.assigned_to, self.admin)` should be enough to test the assertion.

Comment (pbanaszkiewicz) [inline workshops/test/test_event.py]: @narayanaditya95 you're right, somehow I thought that line 343 was doing a request.

Yes, no need to `refresh_from_db()` in this case, and line 347 should be `self.assertEqual(event.assigned_to, self.admin)`.

Comment (narayanaditya95) [inline workshops/test/test_event.py]: Why do we have to `event.refresh_from_db()` when we are fetching the event object after a POST?
`self.assertEqual(event.assigned_to, self.admin)` should be enough to test the assertion.

**Pull request merged**